src/Mod/Fem/femresult/resulttools.py

```diff
@@ -610,13 +641,12 @@ def calculate_von_mises(stress_tensor):
     normal = stress_tensor[:3]
     shear = stress_tensor[3:]
     pressure = np.average(normal)
-    return np.sqrt(1.5 * np.linalg.norm(normal - pressure)**2 + 3.0 * np.linalg.norm(shear)**2)
+    return np.sqrt(1.5 * np.linalg.norm(normal - pressure) ** 2 + 3.0 * np.linalg.norm(shear) ** 2)
 
 
 def calculate_principal_stress_std(
-    stress_tensor
+        stress_tensor
```

Comment:
is this whitespace change necessary?

Reply:
No it is not. PyCharm automatically did this

Reply:
Can you please undo this change? The whitespace is not consistent with it.
(I use Visual Studio and this also sometimes adds faulty whitespace, it seems that no editor is perfect hereby, sadly)

Reply:
done